### PR TITLE
docs: unite boolean in site

### LINKED ISSUE #25056: Unite api(boolean) in site
image

官网 API 中 boolean 出现 2 种表达方式。计划统一一下。

- [x] 1. true

- [ ] 2. `true`

用哪一种？

### PR DESCRIPTION
**Pull request by xrkffgg**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 🔗 Related issue link
close https://github.com/ant-design/ant-design/issues/25056

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |       none    |
| 🇨🇳 Chinese |  none         |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

-----
[View rendered components/affix/demo/basic.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/affix/demo/basic.md)
[View rendered components/button/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/button/index.en-US.md)
[View rendered components/button/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/button/index.zh-CN.md)
[View rendered components/calendar/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/calendar/index.en-US.md)
[View rendered components/calendar/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/calendar/index.zh-CN.md)
[View rendered components/card/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/card/index.en-US.md)
[View rendered components/carousel/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/carousel/index.en-US.md)
[View rendered components/collapse/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/collapse/index.en-US.md)
[View rendered components/collapse/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/collapse/index.zh-CN.md)
[View rendered components/config-provider/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/config-provider/index.en-US.md)
[View rendered components/config-provider/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/config-provider/index.zh-CN.md)
[View rendered components/mentions/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/mentions/index.en-US.md)
[View rendered components/mentions/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/mentions/index.zh-CN.md)
[View rendered components/message/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/message/index.en-US.md)
[View rendered components/message/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/message/index.zh-CN.md)
[View rendered components/modal/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/modal/index.en-US.md)
[View rendered components/modal/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/modal/index.zh-CN.md)
[View rendered components/notification/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/notification/index.en-US.md)
[View rendered components/notification/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/notification/index.zh-CN.md)
[View rendered components/select/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/select/index.en-US.md)
[View rendered components/select/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/select/index.zh-CN.md)
[View rendered components/switch/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/switch/index.en-US.md)
[View rendered components/switch/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/switch/index.zh-CN.md)
[View rendered components/table/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/table/index.en-US.md)
[View rendered components/table/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/table/index.zh-CN.md)
[View rendered components/tabs/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/tabs/index.en-US.md)
[View rendered components/tabs/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/tabs/index.zh-CN.md)
[View rendered components/tag/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/tag/index.en-US.md)
[View rendered components/tag/index.zh-CN.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/tag/index.zh-CN.md)
[View rendered components/timeline/index.en-US.md](https://github.com/ant-design/ant-design/blob/unite-boolean/components/timeline/index.en-US.md)

### CODE REVIEW

File: components/tree/index.zh-CN.md [hunk 1]
```diff
@@ -69,9 +69,9 @@ cover: https://gw.alipayobjects.com/zos/alicdn/Xh-oWqg9k/Tree.svg
 
 ### DirectoryTree props
 
-| 参数         | 说明                                                 | 类型   | 默认值 |
-| ------------ | ---------------------------------------------------- | ------ | ------ |
-| expandAction | 目录展开逻辑，可选 `false` `'click'` `'doubleClick'` | string | click  |
+| 参数         | 说明                                                 | 类型   | 默认值  |
+| ------------ | ---------------------------------------------------- | ------ | ------- |
```
Comment (yoyo837) [inline components/tree/index.zh-CN.md:73]: 强迫症看见你加了一杠 😂

File: components/modal/index.en-US.md [hunk 2]
```diff
@@ -18,7 +18,7 @@ When requiring users to interact with the application, but without jumping to a
 | afterClose | Specify a function that will be called when modal is closed completely. | function | - |
 | bodyStyle | Body style for modal body element. Such as height, padding etc. | CSSProperties | {} |
 | cancelText | Text of the Cancel button | string\|ReactNode | `Cancel` |
-| centered | Centered Modal | Boolean | `false` |
+| centered | Centered Modal | Boolean | false |
```
Comment (fireairforce) [inline components/modal/index.en-US.md:21]: 这里 Boolean 改成小写，下面也一样

File: components/tree/index.zh-CN.md (hunk 1 above)
Comment (xrkffgg) [reply components/tree/index.zh-CN.md:73]: 这个应该是 VSCode 插件自动加的 😂😂

File: components/modal/index.en-US.md (hunk 2 above)
Comment (xrkffgg) [reply components/modal/index.en-US.md:21]: OK

**Comment by xrkffgg** [comment]
大家都可以来找茬~

**Comment by xrkffgg** [comment]
😂  改哭了我要

**Comment by zombieJ** [comment]
提炼一下规范写到这里，以后就以你这个为准了 ：https://github.com/ant-design/ant-design/wiki/API-Naming-rules

**Comment by xrkffgg** [comment]
All CI passed.

**Pull request merged**